drivers/SmartThings/zigbee-window-treatment/src/test/test_zigbee_window_treatment_screen_innovations.lua

```diff
@@ -402,21 +402,4 @@ test.register_coroutine_test(
   end
 )
 
-test.register_coroutine_test(
-    "Power Source test cases",
-    function()
-        test.socket.zigbee:__queue_receive({ mock_device.id,
-        Basic.attributes.PowerSource:build_test_attr_report(mock_device, 3) })
-        test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.powerSource.powerSource.battery()) )
-
-        test.socket.zigbee:__queue_receive({ mock_device.id,
-        Basic.attributes.PowerSource:build_test_attr_report(mock_device, 2) })
-        test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.powerSource.powerSource.dc()) )
-
-        test.socket.zigbee:__queue_receive({ mock_device.id,
-        Basic.attributes.PowerSource:build_test_attr_report(mock_device, 0) })
-        test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.powerSource.powerSource.unknown()) )
-    end
-)
-
```

Comment:
I think you should have lest these tests in. 

The point of my suggested change was that the functionality should be the same from what you wrote.

Reply:
Hi @greens,
Added the integration test case for `powerSource` capability.